### ISSUE TITLE
POT file might be missing from po folder

### ISSUE THREAD

**Issue by scootergrisen** [geany/geany#1786]
The instructions on https://www.geany.org/Contribute/Translators says to run "msginit -l ll_CC -o ll.po -i geany.pot" but there is no geany.pot file in the po folder.

Is this an error?

I see a link to the pot file at the bottom of:
https://i18n.geany.org/

So i guess i just take it from there but perhaps the page should be corrected.

**Comment by codebrainz** [comment]
You can run `make geany.pot` in the `po` directory to generate one.

**Comment by scootergrisen** [comment]
Would it not easier to tell translations to get the pot file at https://i18n.geany.org/ and rename it and change the header a bit if its updated every day?
That way translators dont need to setup msginit and make and stuff.

**Comment by frlan** [comment]
Fair point. On the other hand you will need autotools etc to check your translations on running Geany anyway. So ...  hmmmm...  Not sure.

**Comment by scootergrisen** [comment]
Maybe just add something like "You can also get a daily updated geany.pot from https://i18n.geany.org/geany.pot" on https://www.geany.org/Contribute/Translators.

When i translate other software i find it easier to just get one file. The pot file.
From that i can start a new translation.

When i want to update the translation later on i just get a new pot file and use that to update. With "msgmerge --update" or from inside a GUI program where it can be done from the menu in some tools like Virtaal.

**Comment by codebrainz** [comment]
I don't know which way is better, but I agree one or both methods should be documented on that page.
